src/resources/conf/local.js

```diff
@@ -0,0 +1,47 @@
+const Nightwatch = require("nightwatch");
+const browserstack = require("browserstack-local");
+require("dotenv").config();
+
+let bs_local;
+
+try {
+    const accessKey =
+        process.env.BROWSERSTACK_ACCESS_KEY;
```

Comment:
Keep an || statement for custom values @ansel2000 . Update the README.md file as well

Reply:
Updated local.js and the README.md